api-v1.md

```diff
@@ -61,6 +61,11 @@ Result will only be valid when gameState is `FINISHED`, beforehand it is undefin
 
 playerXSubmitted will switch back to false once the gameState switches to `FINISHED`
 
+### query parameters:
+- showVisualizationData: boolean, optional, default is true
+    - If true, the response will contain the visualization data
+    - If false, the visualization data in the response will be an empty list
```

Comment:
do we really want to return an empty list? That just looks like the data is empty imo
I think I'd prefer an alternate response that doesn't even have the key `visualizationData`

Reply:
That would make handling the response much more annoying, now you'd always have a nullable field etc.
When calling you make the conscious choice of not wanting that data, so you know it's empty because you told the backend to not include the data.

Reply:
I personally disagree. If @0815Sailsman says it's fine, it's fine. But I disagree

Reply:
Frontend team said your version will be easier, so we'll do that